src/paths/projects/supporters.ts

```diff
@@ -28,6 +28,14 @@ ProjectsSupportersRouter.post(
     }
 );
 
+ProjectsSupportersRouter.get(
+    '/believers',
+    rl('ProjectSupportersBelieversFetch', 30, 5),
```

Comment:
This limit will almost always get hit.
Maybe double it, (30,15), 15 requests every 30 seconds.
That would be 1 request every 2, I feel that's fair


I would also set the bucket as the same as the other get request. see how csv and json have the same bucket but different limit time windows? Yes that.


So TLDR:
```suggestion
    rl('ProjectSupportersFetch', 30, 15),
```